packages/logger/tsconfig.json

```diff
@@ -12,21 +13,14 @@
     "declaration": true,
 
     "noImplicitAny": false,
-    "allowSyntheticDefaultImports": false,
+    "allowSyntheticDefaultImports": true,
     "removeComments": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
-
+    "noUnusedParameters": false,
```

Comment:
I don't think this should be turned off. If you wish to have a parameter in these you can prefix it with `_` as what Typescript offers to go around this issue in those edge cases when needed. It's a great tool to see which parameters have been forgotten even though eslint checks it more strict in our current eslint rules I believe